app/uk/gov/hmrc/securemessage/services/SecureMessageService.scala

```diff
@@ -221,14 +221,25 @@ class SecureMessageService @Inject()(
           val _ = auditRetrieveEmail(None)
           Left(CustomerEmailError(customerParticipant, elr))
     }))
-
   @SuppressWarnings(
     Array("org.wartremover.warts.Nothing", "org.wartremover.warts.Any", "org.wartremover.warts.Product"))
   private def sendAlert(receivers: CustomerParticipants, alert: core.Alert)(
     implicit hc: HeaderCarrier,
     ec: ExecutionContext): EitherT[Future, SecureMessageError, Unit] = {
+
+    val enrolmentString = receivers.success.map(_.identifier).headOption.flatMap { identifier =>
+      identifier.enrolment match {
+        case Some(key) => Some(s"$key~${identifier.name}~${identifier.value}")
+        case _         => None
```

Comment:
`identifier.enrolment.map(key => s"$key~${identifier.name}~${identifier.value}")` maybe?

Reply:
I had that before , but then though this is more readable, do you agree ?